src/test/tests_common.h

```diff
@@ -44,6 +44,7 @@
 #include <cart/api.h>
 
 #include "crt_internal.h"
+volatile int	 myflag = 0;
```

Comment:
(style)  do not initialise globals to 0
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt